modules/gatk4/haplotypecaller/main.nf

```diff
@@ -26,23 +26,22 @@ process GATK4_HAPLOTYPECALLER {
     script:
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
-    def interval_option = intervals ? "-L ${intervals}" : ""
-    def dbsnp_option    = dbsnp ? "-D ${dbsnp}" : ""
-    def avail_mem       = 3
+    def interval_command = intervals ? "-L ${intervals}" : ""
+    def dbsnp_command = dbsnp ? "-D ${dbsnp}" : ""
```

Comment:
can we align this?

Reply:
I choose not to align, because I don't think we align prefix and args by default in template

Reply:
ok